addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -8,17 +8,19 @@
 </head>
 <body style="font-family:Verdana, Arial,sans-serif; color: #454748;">
 <t t-set="subtype_internal" t-value="subtype and subtype.internal"/>
+<t t-set="show_header" t-value="has_button_access or email_show_header"/>
+<t t-set="show_footer" t-value="show_header and author_user and email_show_footer"/>
 <!-- HEADER -->
 <t t-call="mail.notification_preview"/>
 <div style="max-width: 900px; width: 100%;">
-<div t-if="has_button_access" itemscope="itemscope" itemtype="http://schema.org/EmailMessage">
+<div t-if="show_header and has_button_access" itemscope="itemscope" itemtype="http://schema.org/EmailMessage">
     <div itemprop="potentialAction" itemscope="itemscope" itemtype="http://schema.org/ViewAction">
         <link itemprop="target" t-att-href="button_access['url']"/>
         <link itemprop="url" t-att-href="button_access['url']"/>
         <meta itemprop="name" t-att-content="button_access['title']"/>
     </div>
 </div>
-<div t-if="subtitles or has_button_access or not is_discussion"
+<div t-if="show_header and (subtitles or has_button_access or not is_discussion)"
```

Comment:
The "not is_discussion" is there I suppose for the "Internal communication" part ? People updating code will still have that part (not doing -u) so they could have new code + old template. In which case it would work. People doing -u won't have the "internal communication" part, and we can probably remove that part of the condition, can't we ?